### ISSUE TITLE
Apply for this grant?

### ISSUE THREAD

**Issue by staltz** [SuperuserLabs/thankful#233]
Hey folks! I like what you're doing, and I stumbled upon this grant which seems highly relevant to this project: [Grant for the Web](https://www.grantfortheweb.org/)

**Comment by Powersource** [comment]
Hehe just saw this, but I think I saw your ssb post around the same time and forwarded it.

I think @ErikBjare is keeping an eye on this (i.e. is subscribed to the newsletter), correct me if I'm wrong. Although at a glance nothing new seems to have happened since september :/

**Comment by ErikBjare** [comment]
Yeah, I introduced myself and Thankful here: https://forum.grantfortheweb.org/t/im-building-activitywatch-time-tracker-and-thankful-browser-extension-for-donating-to-creators/762/8
